google/auth/_default.py

```diff
@@ -112,7 +112,9 @@ def _get_gae_credentials():
 
 
 def _get_gce_credentials():
-    if _metadata.ping():
+    # TODO: Ping now requires a request argument. Figure out how to deal with
```

Comment:
I am big time :-1: on `TODO` in code that doesn't have a corresponding issue filed